src/Microsoft.Identity.Web/DownstreamApiSupport/IDownstreamWebApi.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Interface used to call a downstream web API, for instance from controllers.
+    /// </summary>
+    public interface IDownstreamWebApi
+    {
+        /// <summary>
+        /// Calls the Web API for the user, based on a description of the
+        /// web API in the configuration.
+        /// </summary>
+        /// <param name="optionsInstanceName">Name of the options instance describing the API. There can
+        /// be several configuration namedsections mapped to a <see cref="CalledApiOptions"/>,
+        /// each for one API. You can pass-in null, but in that case <paramref name="calledApiOptionsOverride"/>
+        /// need to be set.</param>
+        /// <param name="calledApiOptionsOverride">Overides the options proposed in the configuration described
+        /// by <paramref name="optionsInstanceName"/>.</param>
+        /// <param name="user">[Optional] claims representing a user. This is useful in scenarios like Blazor
+        /// or Azure Signal R where the HttpContext is not available. In the other scenarios, the library
+        /// will find the user itself.</param>
+        /// <param name="content">Http content in the case where <see cref="CalledApiOptions.HttpMethod"/> is
+        /// <see cref="HttpMethod.Patch"/>, <see cref="HttpMethod.Post"/>, <see cref="HttpMethod.Put"/>.</param>
+        /// <returns>An <see cref="HttpResponseMessage"/> that the application will process.</returns>
+        public Task<HttpResponseMessage> CallWebApiForUserAsync(
```

Comment:
I would reorder parameters here to match `CallWebApiForAppAsync` and other methods:
`optionsInstanceName`, `calledApiOptionsOverride`, `content`, `user`.

Reply:
Done